src/index.js

```diff
@@ -457,6 +545,60 @@ class Searchbase {
     this.setValue(value);
   };
 
+  //mic event
+  onMicClick = () => {
```

Comment:
Why you're not taking `options` as param.
It can have this signature `onMicClick(micOptions: Object, options: Options = defaultOptions)`.
Apply the mic options if defined for e.g language.